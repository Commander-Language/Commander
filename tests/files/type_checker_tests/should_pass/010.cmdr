//Test bindings type check
x: bool = true;
y = x;
