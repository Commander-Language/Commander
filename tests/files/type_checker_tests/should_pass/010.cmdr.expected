(PrgmNode
	(ExprStmtNode (BinOpExprNode (BindingNode x (BoolTypeNode BOOL)) SET (BoolExprNode true BOOL) BOOL))
	(ExprStmtNode (BinOpExprNode (BindingNode y) SET (LValueExprNode (VarExprNode x BOOL) BOOL) BOOL)))
